ExampleBlock.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
Please fix coding standards:


FOUND 3 ERRORS AND 1 WARNING AFFECTING 3 LINES
------------------------------------------------------------------------------------------------------------------------------------------------------------
 30 | ERROR   | [x] Concat operator must be surrounded by a single space
 31 | ERROR   | [x] TRUE, FALSE and NULL must be uppercase; expected "TRUE" but found "true"
 39 | WARNING | [x] There must be no blank line following an inline comment
 39 | ERROR   | [x] No space found before comment text; expected "// $markup .= 'There are 3 articles.';" but found "//$markup .= 'There are 3 articles.';"